my-app/src/components/ContentList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+const ContentList = ({content}) => {
+    return (
+        <div className="cards_box">
+     { content && content.map(content=>(  
+
+          <div class="card">
+            <div class="card-header">
+            {content ?<p><img className="content_img" src={content.urlToImage} alt ="" /> </p> :''}
+            </div>
+            <div class="card-body">
+              {content ?<span className="content_date">  {content.publishedAt} </span> :''}
+        
+              {content ? <h4 className="content_title">{content.title}  </h4>:''}
+        
+              {content ?<p> {content.description.substring(0,200)}... </p> :''}
+        
+              {/* link */}
```

Comment:
If you meant to leave a comment here, you could install some of vscode comment extensions like `better Comments`, it would look much neater.
However, be aware of your comments too. you don't want to end up with an unclean code.